daemon/daemon_unix.go

```diff
@@ -166,6 +166,11 @@ func verifyPlatformContainerSettings(daemon *Daemon, hostConfig *runconfig.HostC
 			return warnings, fmt.Errorf("Invalid value: %v, valid memory swappiness range is 0-100.", swappiness)
 		}
 	}
+	if hostConfig.CPUShares > 0 && !daemon.SystemConfig().CPUShares {
+		warnings = append(warnings, "Your kernel does not support CPU cfs shares. Shares discarded.")
```

Comment:
Can we remove the `cfs` bit? It's pretty linux specific for something that can be shared with other unix-ish platforms.